digdag-guice-rs-server-undertow/src/main/java/io/digdag/guice/rs/server/undertow/UndertowServer.java

```diff
@@ -185,7 +192,15 @@ public static UndertowServerControl start(UndertowServerConfig config, ServerBoo
             if (config.getAccessLogPath().isPresent()) {
                 handler = buildAccessLogHandler(config, handler);
             }
-            appHandler = Handlers.trace(handler);
+            handler = Handlers.trace(handler);
+
+            // support "Accept-Encoding: gzip | deflate" (response content encoding)
+            handler = new EncodingHandler(handler,
```

Comment:
`io.undertow.server.handlers.encoding.EncodingHandler` is to encode a response not a request, right? So ideally speaking, it seems this change should be moved to https://github.com/treasure-data/digdag/pull/591.